include/odp/api/spec/pool_types.h

```diff
@@ -186,6 +186,20 @@ typedef struct odp_pool_capability_t {
 	/** Maximum number of pools of any type (odp_pool_type_t) */
 	uint32_t max_pools;
 
+	/** Pool user area persistence
+	 *
+	 * When supported, implementation does not overwrite event user area
+	 * content at any point of event lifetime nor after freeing an event
+	 * back into pool.
+	 *
+	 * 0: User area content is maintained throughout regular event usage
+	 *    after allocation, but may be modified after free (default)
+	 * 1: User area content is maintained throughout regular event usage
+	 *    and additionally also after event is freed into the pool (between
+	 *    event free and allocation)
+	 */
+	odp_bool_t uarea_persistence;
```

Comment:
This could be defined per pool type. Documentation can be only once e.g. under buffer struct.

Reply:
Now per pool type in v3.